database/tables/questions.pg

```diff
@@ -44,6 +44,7 @@ indexes
     questions_pkey: PRIMARY KEY (id) USING btree (id)
     questions_course_id_number_nondeleted_key: UNIQUE USING btree (course_id, uuid) WHERE deleted_at IS NULL
     questions_course_id_uuid_key: UNIQUE USING btree (course_id, uuid)
+    idx_questions_public_share: USING btree (course_id) WHERE deleted_at IS NULL AND (share_publicly = true OR share_source_publicly = true)
```

Comment:
This index name doesn't follow any of our established conventions.

Reply:
I just copy pasted it from your comment above. 

I know we have conventions for indexes on single columns, what are our conventions for indexes that deal with multiple columns (in this case, 4 columns!).

Reply:
That was example code, not meant to be literally copy/pasted 😅 

It's true that we don't have great conventions for partial indexes. Perhaps `questions_course_id_publicly_shared_idx`? At the very least, we should end with `idx`, not start with it`.

Reply:
sounds good, thanks!

Reply:
done in 3549c7d91